app/controllers/weets.js

```diff
@@ -40,4 +44,23 @@ const getWeets = async (req, res, next) => {
   }
 };
 
-module.exports = { createWeet, getWeets };
+const sendCongratulationsEmail = async () => {
+  try {
+    logger.error('weets-controller::sendCongratulationsEmail::start job :)');
+    const dailyWeets = await weetService.getWittererOfTheDay();
+    if (!dailyWeets) return false;
+    const witterer = await userService.getUserById(dailyWeets.user_id);
+    if (!witterer) return false;
+    const emailSended = await mailerService.sendCongratulationEmail(witterer.dataValues, dailyWeets);
+    logger.error('weets-controller::sendCongratulationsEmail::emailSended::', JSON.stringify(emailSended));
+    return emailSended;
+  } catch (error) {
+    logger.error(`weets-controller::sendCongratulationsEmail::error::${error.message}`);
+    return error;
+  }
+};
+
+const job = new CronJob(CRON_TIME, () => sendCongratulationsEmail(), null, true, TIME_ZONE);
+job.start();
```

Comment:
This is not a code for a controller. I think it would be better in a `scheduledTasks` or `crons` folder, at the same level than the `controllers`, `interactors`, `services`, etc.